lib/src/emoji_picker_extended_utils.dart

```diff
@@ -0,0 +1,72 @@
+import 'dart:convert';
+import 'dart:math';
+
+import 'package:emoji_picker_flutter/emoji_picker_flutter.dart';
+import 'package:emoji_picker_flutter/src/recent_emoji.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+import 'emoji_picker_internal_utils.dart';
+
+/// Helper class that provides extended usage
+
```

Comment:
I prefer to have no space here to keep the style constant between the other files 🙏